src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoGetAbbreviatedEraName.cs

```diff
@@ -11,59 +12,28 @@ namespace System.Globalization.Tests
-            string abbreviatedEraName = DateTimeFormatInfoData.GetAbbreviatedEraName(cultureInfo);
-
-            VerificationHelper(info, 0, abbreviatedEraName);
-            VerificationHelper(info, 1, abbreviatedEraName);
+            yield return new object[] { new CultureInfo("en-US").DateTimeFormat, 0, DateTimeFormatInfoData.EnUSAbbreviatedEraName() };
+            yield return new object[] { new CultureInfo("en-US").DateTimeFormat, 1, DateTimeFormatInfoData.EnUSAbbreviatedEraName() };
+            yield return new object[] { new DateTimeFormatInfo(), 0, "AD" };
+            yield return new object[] { new DateTimeFormatInfo(), 1, "AD" };
+            yield return new object[] { new CultureInfo("ja-JP").DateTimeFormat, 1, DateTimeFormatInfoData.JaJPAbbreviatedEraName() };
         }
 
-        // PosTest2: Call GetAbbreviatedEraName to get Era's abbreviated name on instance created from ctor
-        [Fact]
-        public void PosTest2()
+        [Theory]
+        [MemberData(nameof(GetAbbreviatedEraName_TestData))]
+        public void GetAbbreviatedEraName(DateTimeFormatInfo format, int era, string expected)
         {
-            DateTimeFormatInfo info = new DateTimeFormatInfo();
-
-            VerificationHelper(info, 0, "AD");
-            VerificationHelper(info, 1, "AD");
+            Assert.Equal(expected, format.GetAbbreviatedEraName(era));
         }
 
-        // PosTest3: Call GetAbbreviatedEraName to get Era's abbreviated name on ja-JP culture
         [Fact]
-        public void PosTest3()
-        {
-            CultureInfo cultureInfo = new CultureInfo("ja-JP");
-            DateTimeFormatInfo info = cultureInfo.DateTimeFormat;
-            string expectedEraName = DateTimeFormatInfoData.GetAbbreviatedEraName(cultureInfo);
-
-            VerificationHelper(info, 1, expectedEraName);
-        }
-
-        // NegTest1: ArgumentOutOfRangeException should be thrown when era does not represent a valid era in the calendar specified in the Calendar property
-        [Fact]
-        public void TestInvalidEra()
-        {
-            DateTimeFormatInfo info = new CultureInfo("en-us").DateTimeFormat;
-
-            Assert.Throws<ArgumentOutOfRangeException>(() =>
-            {
-                info.GetAbbreviatedEraName(-1);
-            });
-            Assert.Throws<ArgumentOutOfRangeException>(() =>
-            {
-                info.GetAbbreviatedEraName(2);
-            });
-        }
-
-        private void VerificationHelper(DateTimeFormatInfo info, int era, string expected)
+        public void GetAbbreviatedEraName_Invalid()
         {
-            string actual = info.GetAbbreviatedEraName(era);
-            Assert.Equal(expected, actual);
+            var format = new CultureInfo("en-US").DateTimeFormat;
+            Assert.Throws<ArgumentOutOfRangeException>(() => format.GetAbbreviatedEraName(-1)); // Era < 0
+            Assert.Throws<ArgumentOutOfRangeException>(() => format.GetAbbreviatedEraName(2)); // Era > 1
```

Comment:
I can extract this 2 as a local, and call it `enUSEraCoun`t, then pass `enUSEraCount + 1` instead of 2